CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Font size resetting when Alacritty is moved between screens
 - Limited payload length in clipboard escape (used for Tmux copy/paste)
 - Alacritty not ignoring keyboard events for changing WM focus on X11
+- Alacritty no longer canonicalizes the working directory on Windows
```

Comment:
Alternative message: `Alacritty no longer converts the working directory to a UNC canonical path on Windows`

Reply:
I'm not sure if the user have to care about the canonicalization at all? We removed it exactly because it ***shouldn't*** affect the user.

We no longer convert the path to a weird UNC path, but I'm not sure if most people know what that is (as a Linux dev, I didn't). Though there's nothing wrong with using the technically correct terms as long as everyone can still understand the change.

Maybe something more like this would work?

```
- Remove UNC path prefix from Windows working directory
```

It's not optimal either, since the usage of `UNC` might still be confusing, but it should be a bit better. I also feel like it might be worth mentioning this was a regression, but I'm not certain how to best state that without just bloating up the message unnecessarily.

If you've got any better suggestions, please let me know.

Reply:
How about

"- Regression which added UNC path prefixes to working directories on Windows"

Reply:
Sounds like a winner to me! I'd just make it `working directory` instead of `working directories`. Since I think that more accurately captures that we're talking about the main working directory Alacritty is launched on and not that this is always prefixed to the working directory (it's fixed once you `cd`, afaik).